src/arch/armv8/armv8-a/cpu.c

```diff
@@ -37,7 +37,7 @@ void cpu_arch_profile_standby()
 
 void cpu_arch_profile_powerdown()
 {
-    int64_t err = psci_power_down(PSCI_WAKEUP_IDLE);
+    int32_t err = psci_power_down(PSCI_WAKEUP_IDLE);
```

Comment:
Please provide a bit of rationale behind this decision/change in the commit message.
From this it is not clear why we are doing this.